packages/core-data/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+## v2.0.0 (Unreleased)
+
+- Breaking: `dispatch("core").receiveTerms` has been deprecated. Please use `dispatch("core").receiveEntityRecords` instead.
+- Breaking: `getCategories` resolvers has been deprecated. Please use `getEntityRecords` resolver instead.
```

Comment:
We should get rid of the [last usages](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/categories/edit.js#L206) of `getCategories` and `isRequestingCategories`.